tests/integration/components/sl-textarea-test.js

```diff
@@ -0,0 +1,305 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import { skip } from 'qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-textarea', 'Integration | Component | sl textarea', {
+    integration: true
+});
+
+test( 'Default classes are applied', function( assert ) {
+    this.render( hbs`
+        {{#sl-textarea}}
+        {{/sl-textarea}}
+    ` );
+
+    assert.strictEqual(
```

Comment:
Follow pattern of other component integration tests, such as https://github.com/softlayer/sl-ember-components/blob/EmberCli-1.13.8/tests/integration/components/sl-alert-test.js#L8